src/data/schema/students.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+const studentSchema = new mongoose.Schema({
+  lastname: {
+    type: String,
+    required: true,
+  },
+  firstname: {
+    type: String,
+    required: true, 
+  },
```

Comment:
Add the following fields:
created By:
dateCreated:
updatedBy:
dateUpdated:
active

Reply:
sir ano po yung type ng datecreated at date updated?